File: DAY48/a1.py
class Dog:

    species = "Canis familiaris"

    def __init__(self,name,age):
        self.name = name
        self.age = age

    def bark(self):
        return f"{self.name} say woof!"


my_dog = Dog("Buddy",3)
print(my_dog.bark())

File: DAY48/a1.java
public class a1{

    public static String species = "Canis familiaris";

    String name;
    int age;

    public a1(String name, int age){
        this.name = name;
        this.age = age;

    }

    public String bark(){
        return this.name + " say woof!";
    }

    public static void main(String[] args){
        a1 myDog = new a1("Buddy",3);
        System.out.println(myDog.bark());
    }
}